jobs/JGLOBAL_STAGE_IC

```diff
@@ -53,38 +52,28 @@ err=0
 ###############################################################
 for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
 
-  # Export MEMDIR; need even if empty
-  export MEMDIR
-
   # Declare COMs
   if [[ "${MODE}" = "cycled" && "${RUN}" = "gdas" ]]; then
-    YMD=${current_cycle:0:8} HH=${current_cycle:8:2} declare_from_tmpl COMOUT_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL
-    export COMOUT_ATMOS_ANALYSIS
+    YMD=${current_cycle:0:8} HH=${current_cycle:8:2} declare_from_tmpl -x COMOUT_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL
```

Comment:
`COMOUT_ATMOS_ANALYSIS` didn't use `MEMDIR` before the last change and still doesn't now. It's only declared when cycled and gdas, which creates an empty `MEMDIR_ARRAY` since it doesn't use `MEMDIR`. I'm running a low-res ATM cycled test on WCOSS2 to be sure all is well after the changes I made based on reviewer feedback. Here is the `COMOUT_ATMOS_ANALYSIS` declaration within the `gdasstage_ic` job log:
```
+ JGLOBAL_STAGE_IC[34]: MEMDIR_ARRAY=()
+ JGLOBAL_STAGE_IC[35]: [[ gdas = \e\n\k\f\g\d\a\s ]]
+ JGLOBAL_STAGE_IC[35]: [[ gdas = \g\e\f\s ]]
+ JGLOBAL_STAGE_IC[46]: MEMDIR_ARRAY+=("")
+ JGLOBAL_STAGE_IC[50]: err=0
+ JGLOBAL_STAGE_IC[53]: for MEMDIR in "${MEMDIR_ARRAY[@]}"
+ JGLOBAL_STAGE_IC[56]: [[ cycled = \c\y\c\l\e\d ]]
+ JGLOBAL_STAGE_IC[56]: [[ gdas = \g\d\a\s ]]
+ JGLOBAL_STAGE_IC[57]: YMD=20211220
+ JGLOBAL_STAGE_IC[57]: HH=18
+ JGLOBAL_STAGE_IC[57]: declare_from_tmpl -x COMOUT_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL
+ bash_utils.sh[35]: [[ NO == \N\O ]]
+ bash_utils.sh[35]: set +x
declare_from_tmpl :: COMOUT_ATMOS_ANALYSIS=/lfs/h2/emc/ptmp/kate.friedman/comrot/testcycstage/gdas.20211220/18//analysis/atmos
```